src/reducers/quiz.js

```diff
@@ -0,0 +1,28 @@
+import quizData from '../data/quiz.json';
+import * as constants from '../constants/quiz';
+
+const initialState = {
+  tests: quizData,
+  selectedTest: {},
+  selectedQuestion: null,
+  selectedAnswer: { selectedAnswer: null, correctAnswer: null },
```

Comment:
Can you use a better name for the variable than `store.selectedAnswer.selectedAnswer`?
Do you really need this variable?

Reply:
This is my mistake: this variable is not used and I need to remove them. Update in next commit